### PR TITLE
Upload report file to media first

### PR DESCRIPTION
**Pull request by bdjilka**

### CODE REVIEW

File: executor/utils.py [hunk 1]
```diff
@@ -109,13 +110,23 @@ def get_user_agent():
     }
 
 
-def upload_file(client, report_name, report_id):
+def upload_file(client, report_name, report_id, owner_id):
     report_filename = os.path.basename(report_name)
-    return client.ns('reporting').reports[report_id].action('upload').post(
-        data=open(report_name, 'rb'),
+    response = requests.post(
```
Comment (marcserrat) [inline executor/utils.py:115]: why not using the client??? you can do the same without requests :)

File: executor/executor.py [hunk 2]
```diff
@@ -60,8 +61,8 @@ def start():
 
     if result:  # pragma: no branch
         try:
-            upload_file(client, result, report_env["report_id"])
-        except (ClientError, Exception) as e:
+            upload_file(client, result, report_env["report_id"], report_to_execute['owner']['id'])
+        except (ClientError, Exception, requests.exceptions.HTTPError) as e:
```
Comment (marcserrat) [inline executor/executor.py:65]: if yoload is done using client.....you can avoid requessts

File: executor/utils.py (hunk 1 above)
Comment (bdjilka) [reply executor/utils.py:115]: I tried and wanted.... but broke my head trying to figure out the final expression

Comment (bdjilka) [reply executor/utils.py:115]: Like you get` client.ns('media').folders['reports_report_file'] ` and then I'm stuck

Comment (ffaraone) [reply executor/utils.py:115]: @bdjilka try:

`client.ns('media').ns('folders').collection('reports_report_file')[owner_id].action('files').post(files=...., headers=...)`

And send only content-type and content-disposition headers.

**Review by ffaraone — CHANGES_REQUESTED** [review]

**Review by ffaraone — APPROVED** [review]

**Review by zzzevaka — APPROVED** [review]

**Pull request merged**